dashboard/app/controllers/peer_reviews_controller.rb

```diff
@@ -14,7 +14,7 @@ def show
   end
 
   def pull_review
-    script = Script.find_by_name(params[:script_id])
+    script = Script.get_from_cache(params[:script_id])
```

Comment:
Agreed. This comes from the routes.rb file. Might be a nasty fix but it should be done